src/main/kotlin/no/nav/klage/oppgave/api/view/KvalitetsvurderingView.kt

```diff
@@ -0,0 +1,10 @@
+package no.nav.klage.oppgave.api.view
+
+data class KvalitetsvurderingView(
+    val grunn: Int?,
+    val eoes: Int?,
```

Comment:
En litt interessant ting er at vi her sender id'en til enumen frem og tilbake. Men i den gamle koden for OppgaveView (som egentlig er backet av Klagebehandling nå), så sender vi navn'et til enumen frem og tilbake. At vi gjør det på to ulike måter er ikke så veldig heldig, så det bør harmoniseres. Og da IMHO til å bruke id. Jeg nevnte dette i en tråd på Slack også..

Reply:
Jeg er jammen ikke helt sikker på hva jeg tenker her. Men har ikke så mye å si så lenge FE og BE er i synk med kodeverket. La oss bruke id inntil videre :)